keyboards/keycapsss/kimiko/rev2/info.json

```diff
@@ -0,0 +1,202 @@
+            { "matrix": [4, 2], "x": 53, "y": 51, "flags": 1 },
+            { "matrix": [4, 1], "x": 40, "y": 47, "flags": 1 },
+            { "matrix": [4, 0], "x": 26, "y": 51, "flags": 1 },
+            { "matrix": [6, 5], "x": 158, "y": 5, "flags": 4 },
+            { "matrix": [6, 4], "x": 171, "y": 4, "flags": 4 },
+            { "matrix": [6, 3], "x": 184, "y": 0, "flags": 4 },
+            { "matrix": [6, 2], "x": 198, "y": 4, "flags": 4 },
+            { "matrix": [6, 1], "x": 211, "y": 11, "flags": 4 },
+            { "matrix": [6, 0], "x": 224, "y": 11, "flags": 4 },
+            { "matrix": [7, 0], "x": 224, "y": 23, "flags": 4 },
+            { "matrix": [7, 1], "x": 211, "y": 23, "flags": 4 },
+            { "matrix": [7, 2], "x": 198, "y": 15, "flags": 4 },
+            { "matrix": [7, 3], "x": 184, "y": 12, "flags": 4 },
+            { "matrix": [7, 4], "x": 171, "y": 15, "flags": 4 },
+            { "matrix": [7, 5], "x": 158, "y": 17, "flags": 4 },
+            { "matrix": [8, 5], "x": 158, "y": 29, "flags": 4 },
+            { "matrix": [8, 4], "x": 171, "y": 27, "flags": 4 },
+            { "matrix": [8, 3], "x": 184, "y": 24, "flags": 4 },
+            { "matrix": [8, 2], "x": 198, "y": 27, "flags": 4 },
+            { "matrix": [8, 1], "x": 211, "y": 35, "flags": 4 },
+            { "matrix": [8, 0], "x": 224, "y": 35, "flags": 4 },
+            { "matrix": [9, 0], "x": 224, "y": 47, "flags": 1 },
+            { "matrix": [9, 1], "x": 211, "y": 47, "flags": 4 },
+            { "matrix": [9, 2], "x": 198, "y": 39, "flags": 4 },
+            { "matrix": [9, 3], "x": 184, "y": 36, "flags": 4 },
+            { "matrix": [9, 4], "x": 171, "y": 39, "flags": 4 },
+            { "matrix": [9, 5], "x": 158, "y": 41, "flags": 4 },
+            { "matrix": [11, 5], "x": 144, "y": 48, "flags": 4 },
+            { "matrix": [10, 5], "x": 130, "y": 52, "flags": 4 },
+            { "matrix": [10, 4], "x": 137, "y": 64, "flags": 4 },
+            { "matrix": [10, 3], "x": 151, "y": 60, "flags": 1 },
+            { "matrix": [10, 2], "x": 171, "y": 51, "flags": 4 },
+            { "matrix": [10, 1], "x": 184, "y": 47, "flags": 1 },
+            { "matrix": [10, 0], "x": 198, "y": 51, "flags": 1 }
+        ],
+        "led_count": 62,
+        "max_brightness": 80,
+        "split_count": [31, 31]
+    },
+    "rgblight": {
+        "led_count": 62,
+        "max_brightness": 80,
+        "split": true,
+        "split_count": [31, 31]
+    },
+    "split": {
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "F5", "pin_b": "F4"}
+                ]
+            }
+        },
+        "matrix_pins": {
+            "right": {
+                "cols": ["F6", "F7", "B1", "B3", "B2", "B6"],
+                "rows": ["D4", "C6", "D7", "E6", "B4", "B5"]
+            }
+        },
+        "soft_serial_pin": "D2",
+        "transport": {
```

Comment:
The addition of this transport block to info.json breaks the code. I will have a look for the correct syntax but it may be best just to remove it for now.

Reply:
removed  transport matrix_state from info.json and compiling works again.